numpy/lib/_shape_base_impl.py

```diff
@@ -1241,16 +1244,29 @@ def tile(A, reps):
 
     >>> b = np.array([[1, 2], [3, 4]])
     >>> np.tile(b, 2)
+    array([[1, 2, 1, 2],
+           [3, 4, 3, 4]])
+    >>> np.tile(b, 2, axis=-1)
     array([[1, 2, 1, 2],
            [3, 4, 3, 4]])
     >>> np.tile(b, (2, 1))
+    array([[1, 2],
+           [3, 4],
+           [1, 2],
+           [3, 4]])
+    >>> np.tile(b, 2, axis=0))
     array([[1, 2],
            [3, 4],
            [1, 2],
            [3, 4]])
 
     >>> c = np.array([1,2,3,4])
-    >>> np.tile(c,(4,1))
+    >>> np.tile(c, (4, 1))
```

Comment:
Maybe just replace this example with `np.tile(c, 4, axis=0)` - which really is clearer!